x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_executed_details.tsx

```diff
@@ -11,108 +11,110 @@ import { pure } from 'recompose';
-        excluded: false,
-        kqlQuery: '',
-        queryMatch: {
-          field,
-          value: escapeQueryValue(queryValue ? queryValue : value),
-        },
-      }}
-      render={(dataProvider, _, snapshot) =>
-        snapshot.isDragging ? (
-          <DragEffects>
-            <Provider dataProvider={dataProvider} />
-          </DragEffects>
-        ) : (
-          <>{value}</>
-        )
-      }
-    />
-  ) : null
-);
-
 export const AuditdExecutedCommandLine = pure<{
   id: string;
   hostName?: string | null;
   userName?: string | null;
+  primary?: string | null;
+  secondary?: string | null;
   processName?: string | null;
   processTitle?: string | null;
   workingDirectory?: string | null;
   args?: string | null;
-}>(({ id, hostName, userName, processName, processTitle, workingDirectory, args }) => (
-  <Details>
-    <EuiFlexGroup justifyContent="flexStart" gutterSize="none">
-      <EuiFlexItem grow={false}>
-        <DraggableAuditdExecutedElement id={id} field="user.name" value={userName} />
-      </EuiFlexItem>
-      <EuiFlexItem grow={false}>@</EuiFlexItem>
-      <EuiFlexItem grow={false}>
-        <DraggableAuditdExecutedElement id={id} field="host.name" value={hostName} />
-      </EuiFlexItem>
-      <MarginRightFlexItem grow={false}>:</MarginRightFlexItem>
-      <EuiFlexItem grow={false}>
-        <DraggableAuditdExecutedElement
-          id={id}
-          field="process.working_directory"
-          value={workingDirectory}
-        />
-      </EuiFlexItem>
-      <EuiFlexItem grow={false}>></EuiFlexItem>
-      <MarginLeftFlexItem grow={false}>
-        <DraggableAuditdExecutedElement id={id} field="process.name" value={processName} />
-      </MarginLeftFlexItem>
-      <MarginLeftFlexItem grow={false}>
-        <DraggableAuditdExecutedElement
-          id={id}
-          field="process.title"
-          queryValue={processTitle != null ? processTitle : ''}
-          value={args}
-        />
-      </MarginLeftFlexItem>
-    </EuiFlexGroup>
-  </Details>
-));
+  session?: string | null;
+}>(
+  ({
+    id,
+    hostName,
+    userName,
+    primary,
+    secondary,
+    processName,
+    processTitle,
+    workingDirectory,
+    args,
+    session,
+  }) => (
+    <Details>
+      <EuiFlexGroup justifyContent="center" gutterSize="none" wrap={true}>
+        <TokensFlexItem grow={false}>Session</TokensFlexItem>
+        <TokensFlexItem grow={false}>
+          <DraggableBadge
+            id={`auditd-loggedin-${id}`}
+            field="auditd.session"
+            value={session}
+            iconType="number"
+          />
+        </TokensFlexItem>
+        <TokensFlexItem grow={false}>
+          <PrimarySecondaryUserInfo
+            id={`auditd-executed-element-${id}`}
+            userName={userName}
+            primary={primary}
+            secondary={secondary}
+          />
+        </TokensFlexItem>
+        <TokensFlexItem grow={false}>@</TokensFlexItem>
```

Comment:
Should the static text like `@` `in`, have an "exists" check for whatever is immediately described after the static text, such that the text doesn't describe the wrong thing when the field is `null` or `undefined`?

For example, if working directory is `null` or `undefined`, then it could read "in executed".